rules/prefer-node-remove.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+const getDocsUrl = require('./utils/get-docs-url');
+
+const getMethodName = callee => {
+	const {property} = callee;
+
+	if (property.type === 'Identifier') {
+		return property.name;
+	}
+
+	return null;
+};
+
+const getCallerName = callee => {
+	const {object} = callee;
+
+	if (object.type === 'Identifier') {
+		return object.name;
+	}
+
+	if (object.type === 'MemberExpression') {
+		const {property} = object;
+
+		if (property.type === 'Identifier') {
+			return property.name;
+		}
+	}
+
+	return null;
+};
+
+const getArgumentName = args => {
+	const [identifier] = args;
+
+	if (identifier.type === 'ThisExpression') {
+		return 'this';
+	}
+
+	if (identifier.type === 'Identifier') {
+		return identifier.name;
+	}
+
+	return null;
+};
```

Comment:
Are any of these methods maybe generally useful and generic enough to be moved to a `util.js` file?

Reply:
Yes, I think `getMethodName` and `getCallerName` would generally be useful. `getArgumentName` only works for a single argument to a function, so I don't think that one would be used much.

Reply:
Would be cool if you could extract them in a separate PR.